src/main/java/org/elasticsearch/index/query/TemplateQueryParser.java

```diff
@@ -52,9 +50,9 @@
 
     private final static Map<String,ScriptService.ScriptType> parametersToTypes = new HashMap<>();
     static {
-        parametersToTypes.put("query",ScriptService.ScriptType.INLINE);
-        parametersToTypes.put("file",ScriptService.ScriptType.FILE);
-        parametersToTypes.put("id",ScriptService.ScriptType.INDEXED);
+        parametersToTypes.put("query", ScriptService.ScriptType.INLINE);
+        parametersToTypes.put("file", ScriptService.ScriptType.FILE);
+        parametersToTypes.put("id", ScriptService.ScriptType.INDEXED);
```

Comment:
Probably not something for this PR but we now have a convention of using `script` for inline, `script_file` for file and `script_id` for indexed scripts (using the ScriptParameterParser to maintain the consistency). Should we follow that convention here? I can see we may want to keep the inline as `query` but maybe we should also support `script` for inline? and `file` and `id` can probably be replaced with `script_file` and `script_id`?